db/migrate/20181213214340_create_project_dependent_repositories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjectDependentRepositories < ActiveRecord::Migration[5.2]
+  def change
+    create_view :project_dependent_repositories, materialized: true
```

Comment:
Can this create the view without data and then let the data get filled in on the first run of the refresh later? That lets us decouple deployment (and having *other* deployments occur) from adding the initial view